docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
+---
+
+# Infrastructure Architecture
+
+JHub Apps, also known as JupyterHub Apps Launcher, is an [Externally-Managed JupyterHub service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html#externally-managed-services) designed to enable users to launch various server types including API services, dashboards such as Panel, Bokeh, Streamlit, standard JupyterLab instances, and any generic Python command.
+
+The system is built on a FastAPI framework as detailed in the [services.apps module](https://github.com/nebari-dev/jhub-apps/blob/main/jhub_apps/service/app.py). It initializes and runs through a [Uvicorn](https://www.uvicorn.org/) server (a WSGI HTTP server), managed by the JupyterHub service manager upon startup:
+
+<div style={{'border-radius': '15px', 'border': '2px solid var(--ifm-color-primary)', 'padding': '20px' }}>
+```python title="jhub_apps/configuration.py#L42-L5"
+c.JupyterHub.services.extend(
+    [
+        {
+            "name": fast_api_service_name,
+            "url": f"http://{c.JAppsConfig.hub_host}:10202",
+            "command": [
+                c.JAppsConfig.python_exec,
+                "-m",
+                "uvicorn",
+                "jhub_apps.service.app:app",
+                "--port=10202",
+                "--host=0.0.0.0",
+                f"--workers={c.JAppsConfig.service_workers}",
+            ],
+            ...
+        }
+    ]
+)
+```
+</div>
+
+This service operates on the Hub's host at port 10202, with a set number of workers as specified in the `service_workers` configuration parameter. It is accessible to users through the Hub's proxy, which directs requests to this service URL.
+
+JupyterHub comprises four primary subsystems:
+
+- a Hub (tornado process) that serves as the core of JupyterHub,
+- a configurable http proxy (node-http-proxy) that routes incoming browser requests,
+- multiple single-user Jupyter notebook servers (Python/IPython/tornado) managed by Spawners,
+- and an authentication module that governs user access.
+
+Upon its launch, Jhub-apps is integrated into JupyterHub as a new service. This addition modifies the Hub's homepage to feature the service's URL, thereby offering users not only access to other services but also control over spawning different frameworks beyond JupyterLab, like Streamlit and Bokeh, while utilizing JupyterHub's authentication and spawner mechanisms.
```

Comment:
```suggestion
When JHub Apps is launched, it is integrated into JupyterHub as a new service. This addition modifies the Hub's homepage to feature the service's URL, thereby offering users not only access to other services but also control over spawning different frameworks beyond JupyterLab, like Streamlit and Bokeh, while utilizing JupyterHub's authentication and spawner mechanisms.
```

Need to be consistent across this document - `JHub Apps` is used when talking about the project as a whole, `jhub-apps` is used when talking about the package.